src/NServiceBus.Gateway.AcceptanceTests/When_sending_a_message_to_multiple_sites.cs

```diff
@@ -0,0 +1,117 @@
+﻿namespace NServiceBus.Gateway.AcceptanceTests
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using NUnit.Framework;
+
+    public class When_sending_a_message_to_multiple_sites : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_be_able_to_reply_to_messages_from_all_sites()
+        {
+            var context = await Scenario.Define<Context>()
+                .WithEndpoint<Headquarters>(b => b.When(async (bus, c) => await bus.SendToSites(new[]
```

Comment:
tbh I find it a bit confusing that a send operation can send to multiple destinations. Normal sends cannot do that.